cookiecutter/main.py

```diff
@@ -25,7 +25,7 @@
 def cookiecutter(
         template, checkout=None, no_input=False, extra_context=None,
         replay=False, overwrite_if_exists=False, output_dir='.',
-        config_file=None, default_config=False, password=None):
+        config_file=None, default_config=False, password=None, dir=None):
```

Comment:
Let's not shadow the [`dir` built-in function](https://docs.python.org/3.7/library/functions.html#dir) please?

Reply:
Ah, I didn't catch that...  This change was made in above PR (https://github.com/cookiecutter/cookiecutter/pull/1224/commits/5a6982dc19330a111ce08075cbf84663ab797284)
Shall I change it back to `subdir`?

Reply:
Better to make it directory. (Full name)
I saw this @michaeljoseph note, but have not time to change it yet. So if you can make it directory it will be great.